{"version":3,"sources":["../../../../source/esmodule/library/error/worker-client-internal-error.js"],"names":[],"mappings":"AAAA,SAAS,yBAAT,QAA0C,mCAA1C;;AAEA,MAAM,yBAAN,SAAwC,yBAAxC,CAAkE;;AAEhE,EAAA,WAAW,CAAC,aAAD,EAAgB;AACzB,UAAO,sDAAqD,aAAa,CAAC,OAAQ,IAAlF;AACA,SAAK,cAAL,GAAsB,aAAtB;AACD;;AAEgB,MAAb,aAAa,GAAG;AAClB,WAAO,KAAK,cAAZ;AACD,GAT+D;;;;AAalE,SAAS,yBAAT","file":"worker-client-internal-error.js","sourcesContent":["import { WorkerClientRejectedError } from './worker-client-rejected-error.js'\n\nclass WorkerClientInternalError extends WorkerClientRejectedError {\n\n  constructor(internalError) {\n    super(`The server exited with the internal error message '${internalError.message}'.`)\n    this._internalError = internalError\n  }\n\n  get internalError() {\n    return this._internalError\n  }\n\n}\n\nexport { WorkerClientInternalError }\n"]}